### PR TITLE
[DDI-474] Add detail that mentions SR is instrumented per project

### PR DESCRIPTION
**Pull request by markzegarelli**
# Amplitude Developer Docs PR

## Description

Describe your changes. 

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

**Comment by katie-amplitude** [comment]
Looks good although one recommendation would be to note that sample rate is also relevant here - 

Session Replay instrumentation happens in the context of an Amplitude Project. Your replay quota is defined on the Organization level. As a result, you may have multiple Session Replay implementations, across multiple projects each with their own sample rate, that pull from the same quota.

File: includes/session-replay/instrumentation-level.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+!!!info Session Replay instrumentation
+    Session Replay instrumentation happens in the context of an Amplitude Project. Your replay quota is defined on the Organization level. As a result, you may have multiple Session Replay implementations, across multiple projects, that pull from the same quota.
```
Comment (markzegarelli) [inline includes/session-replay/instrumentation-level.md:2]:
```suggestion
    Session Replay instrumentation happens in the context of an Amplitude Project. Your replay quota is defined on the Organization level. As a result, you may have multiple Session Replay implementations, across multiple projects each with their own sample rate, that pull from the same quota.
```

**Comment by markzegarelli** [comment]
Updated.

**Review by katie-amplitude — APPROVED** [review]
thank you!

**Pull request merged**